### ISSUE TITLE
Please stop tracking output 

### ISSUE THREAD

**Issue by deejoe** [Lugor/lugor.github.io-src#1]
the output directory is a bit like the *pyc files: It is generated content, not source, and so is inappropriate to try to version. 

One level up from the output directory, please issue:

```
git rm -r --cache output
```

and then commit the change so that don't have to keep working around  these files.

**Comment by deejoe** [comment]
@cprofitt 

We'll probably also want to add `output` to our .gitignore

**Comment by deejoe** [comment]
I guess I should say that the output files are fine to track, but maybe tracking it separately would make it more convenient, as has already been established with http://github.com/lugor/lugor.github.io

**Comment by cprofitt** [comment]
I followed some directions to set this up:
https://fedoramagazine.org/make-github-pages-blog-with-pelican/

Have not looked at the details much. Ran both the commands you requested.

**Comment by deejoe** [comment]
On Mon, Sep 05, 2016 at 03:27:50PM -0700, cprofitt wrote:

> I followed some directions to set this up:
> https://fedoramagazine.org/make-github-pages-blog-with-pelican/
> 
> Have not looked at the details much. Ran both the commands you requested.

Aha, have just given that a quick look. I guess it tries to handle the `output` directory separately by treating it as a submodule.

This confuses me, though, because when I cloned the -src repository, I got the contents of `output` .

From the [Pro Git book](https://git-scm.com/docs/git-submodule):

> When cloning or pulling a repository containing submodules however, these
> will not be checked out by default; the init and update subcommands will
> maintain submodules checked out and at appropriate revision in your working
> tree.

So maybe the `git submodule add` part of that didn't work as expected?

In any case, I did a fresh clone and `output` doesn't come down, so that
works for me, for now. We can try to sort out the submodule stuff later, now that
we're clear that that's what we were trying to do.

For now, this closes #1

Thanks!
